src/collection.js

```diff
@@ -322,7 +323,38 @@ export default class Collection {
   listRecords(options={}) {
     const path = endpoint("record", this.bucket.name, this.name);
     const reqOptions = this._collOptions(options);
-    return this.client.paginatedList(path, options, reqOptions);
+    if (options.hasOwnProperty("at")) {
+      return this._getSnapshot(options.at);
+    } else {
+      return this.client.paginatedList(path, options, reqOptions);
+    }
+  }
+
+  _getSnapshot(at) {
+    if (!Number.isInteger(at) || at <= 0) {
+      throw new Error("Invalid argument, expected a positive integer.");
+    }
+    return this.bucket.listHistory({
```

Comment:
We could add a XXX comment  that explains that we take the history forward, and that it would more efficient (and more chance to be complete) if we'd do it backward